packages/python-instagram-private-api/PKGBUILD

```diff
@@ -0,0 +1,28 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-instagram-private-api
+_pkgname=instagram_private_api
+pkgver=1.6.0.0
+pkgrel=1
+pkgdesc='A Python wrapper for the Instagram private API with no 3rd party dependencies.'
+arch=('any')
+url='https://pypi.org/project/instagram-private-api'
+license=('MIT')
+depends=()
+makedepends=('python-setuptools')
+options=(!emptydirs)
+source=("https://files.pythonhosted.org/packages/source/${_pkgname::1}/$_pkgname/$_pkgname-$pkgver.tar.gz")
+sha512sums=('SKIP')
+
+build() {
+  cd "$_pkgname-$pkgver"
+
+  python setup.py build
+}
+
+package() {
+  cd "$_pkgname-$pkgver"
+
+  python setup.py install --root="$pkgdir" --prefix=/usr -O1 --skip-build
```

Comment:
This is just a comment to tell you that you can test your PKGBUILD in a clean sandbox with `ba-dev` cf. https://gist.github.com/noraj/5045be32490f1161161e16fc5820066c#testing

E.G.

```
$ ba-dev -b
$ ba-dev -e 'python' -p python-instagram-private-api-1.6.0.0-1-any.pkg.tar.zst
Package python-instagram-private-api-1.6.0.0-1-any.pkg.tar.zst installed correctly! Testing it now...
Python 3.10.9 (main, Dec 19 2022, 17:35:49) [GCC 12.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from instagram_private_api import Client, ClientCompatPatch
```

Reply:
Does it work only with packages of python tools or with all packages?

Reply:
Testing all packages :)